clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -289,6 +430,7 @@ private void releasePool(String poolType, String poolName, ResourcePageSource re
     ServerSideConfiguration.Pool pool = resourcePageSource.getPool();
     OffHeapResource source = services.getService(OffHeapResourceIdentifier.identifier(pool.getServerResource()));
     if (source != null) {
+      unRegisterPoolStatistics(resourcePageSource);
```

Comment:
Just to clarify this is the call stack for calling unRegisterPoolStatistics(...);
1. EhcacheActiveEntity.destroy()
2. ehcacheStateService.destroy()
3. EhcacheStateServiceImpl.releasePools(...)
4. EhcacheStateServiceImpl.releasePool(...)
5. EhcacheStateServiceImpl.unRegisterPoolStatistics(...);